static/app/types/event.tsx

```diff
@@ -414,6 +420,7 @@ export interface EventTransaction
   entries: (EntrySpans | EntryRequest)[];
   startTimestamp: number;
   type: EventOrGroupType.TRANSACTION;
+  performanceDetectorData?: Record<string, PerformanceDetectorData>;
```

Comment:
Looks like this data will not be stored on `event`s anymore. Once discussions around that are finalized, I'll update accordingly

Reply:
...or will it? 😳